CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## Unreleased
+
+### Fixed
+- `filter-dropdown-remove:after` no longer positioned absolutely
+
+### Modified
+- `btn-clear` has bg white, rathe than transparent for ACAG on light grays.
```

Comment:
"has a white background, rathe*r* than transparent in order to pass *WCAG* on light grays"